schema/src/schema.json

```diff
@@ -27,6 +27,17 @@
           },
           "response": {
             "$ref": "http://json-schema.org/draft-07/schema#"
+          },
+          "minimumSupportAppVersion": {
+            "type": "object",
+            "properties": {
+              "ios": {
+                "type": "string"
```

Comment:
How about this?

```suggestion
                "$ref": "https://raw.githubusercontent.com/microsoft/json-schemas/main/spfx/semver.schema.json"
```

Reply:
https://raw.githubusercontent.com/microsoft/json-schemas/main/spfx/semver.schema.json